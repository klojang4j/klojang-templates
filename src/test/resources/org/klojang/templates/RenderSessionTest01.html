<html>
<head>
<title>~%title%</title>
</head>
<body>
	<table>
		<thead>
			<tr>
				<th>Company Name</th>
				<th>PO Box</th>
				<th>Established</th>
				<th>Director</th>
				<th>Department</th>
			</tr>
		</thead>
		<tbody>
			~%%begin:company%<tr>
				<td>~%director%</td>
				<td>
					<table>~%%begin:departments%
						<tr><td>~%name%</td><td>~%manager%</td></tr>~%%end:departments%
					</table>
				</td>
			</tr>~%%end:company%
		</tbody>
	</table>
</body>
</html>