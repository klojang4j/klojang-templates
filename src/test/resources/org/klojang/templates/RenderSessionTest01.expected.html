<html>
<head>
<title>Hello World!</title>
</head>
<body>
	<table>
		<thead>
			<tr>
				<th>Company Name</th>
				<th>PO Box</th>
				<th>Established</th>
				<th>Director</th>
				<th>Department</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>John Smith</td>
				<td>
					<table>
						<tr><td>HR</td><td>Kristina Aguilera</td></tr>
						<tr><td>ICT</td><td>Woody Harrelson</td></tr>
						<tr><td>Sales</td><td>Robert de Niro</td></tr>
					</table>
				</td>
			</tr>
		</tbody>
	</table>
</body>
</html>